src/main/java/crazypants/enderio/machine/vacuum/TileVacuumChest.java

```diff
@@ -191,9 +226,86 @@ public boolean isItemValidForSlot(int var1, ItemStack var2) {
     return true;
   }
 
+  public boolean isItemValidForFilter(ItemStack itemstack) {
+    return itemstack != null && itemstack.getItem() == EnderIO.itemBasicFilterUpgrade && itemstack.getItemDamage() == 0;
+  }
+
+  public int getRange() {
+    return range;
+  }
+
+  private int limitRange(int range) {
+    return Math.max(1, Math.min(Config.vacuumChestRange, range));
```

Comment:
Maybe 0 range should be possible? Not sure how useful that would be but it seems unnecessary to floor it at 1 :P